DashAI/back/dataloaders/classes/json_dataloader.py

```diff
@@ -71,13 +72,14 @@ def load_data(
             dataset = load_dataset("json", data_files=url, field=field)
         elif file:
             files_path = self.extract_files(dataset_path, file)
-            try:
-                if files_path.split("/")[-1] == "files":
+            if files_path.split("/")[-1] == "files":
```

Comment:
This operation apparently repeats itself quite a lot, wouldn't it be better to implement it as a method that is inherited from `BaseDataloader`?

Reply:
I think it would be better not to generalize it yet in BaseDataloader since only 2 of the 3 have a similar behavior, the imagedataloader is different since it can only come in ZIP format, so it changes the code.

Reply:
Ok!